LeaderboardBackend/Properties/launchSettings.json

```diff
@@ -1,14 +1,14 @@
 {
   "profiles": {
-    "LeaderboardBackendCsPoc": {
+    "LeaderboardBackend": {
       "commandName": "Project",
       "launchBrowser": true,
       "launchUrl": "swagger",
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
       },
       "dotnetRunMessages": true,
-      "applicationUrl": "https://localhost:7128;http://localhost:5240"
+      "applicationUrl": "https://localhost:8080;http://localhost:8000"
```

Comment:
Could you edit the readme, too? There's a line near the bottom where I wrote the URL out explicitly.

Reply:
Edited (the urls were actually not needed)